modules/products/bitbucket/variables.tf

```diff
@@ -128,3 +128,8 @@ variable "elasticsearch_replicas" {
   }
 }
 
+variable "display_name" {
+  description = "The display name of Bitbucket instance."
+  type        = string
+  default     = null
+}
```

Comment:
Do we have any restriction for display name, such as length or any special character? In this case it would be good to add validation here or in root level (preferably).

Reply:
That's actually a good point. I rechecked Bitbucket Helm chart, there is no length restriction on this field.. So I guess for now, the `string` type restriction should be enough?